labelSelector.js

```diff
@@ -164,6 +164,29 @@ LabelSelector.prototype.covers = function(selector) {
   return true;
 };
 
+// Exports the labelSelector as a string in the API format, exports as matchExpressions
+LabelSelector.prototype.exportJSON = function() {
+  var REVERSE_OPERATOR_MAP = {
+    "in": "In",
+    "not in": "NotIn",
+    "exists": "Exists",
+    "does not exist": "DoesNotExist"
+  };
```

Comment:
Might be better to keep this beside OPERATOR_MAP so it's obvious if you change one you need to change the other.